3d/splitflap.scad

```diff
@@ -248,7 +254,7 @@ module m4_captive_nut(bolt_length=m4_bolt_length) {
 
 // ##### Struts for bracing spool #####
 module spool_strut_tab_hole(narrow) {
-    square([thickness, narrow ? spool_strut_tab_width_narrow : spool_strut_tab_width], center=true);
+    square([thickness + spool_tab_clearance, narrow ? spool_strut_tab_width_narrow + spool_tab_clearance : spool_strut_tab_width + spool_tab_clearance], center=true);
```

Comment:
Hmmm, I wonder if it would make sense to define the clearance values not on a per-part basis, but as 2 high level parameters, something like:
- thickness_clearance
- cut_clearance

Since cuts should be consistent and much lower variability than thickness, it seems like it would make sense to use a smaller clearance value for cut-to-cut joints (where dimensions are entirely controlled by cuts) compared to cut-to-thickness joints (where dimensions rely on inherent material thickness and therefore need a larger tolerance to ensure fit).

This part of the model involves both, so it would change to something like:
```suggestion
    square([thickness + thickness_clearance, narrow ? spool_strut_tab_width_narrow + cut_clearance : spool_strut_tab_width + cut_clearance], center=true);
```

Reply:
Maybe, but the variability for both is wide enough that I'm not sure making the distinction would be beneficial. Both the kerf and the thickness are precise but not accurate, so assuming that the initial values are correct the tolerances should be roughly the same for both. The bigger overarching problem is that the design relies on friction-fit parts which necessitate accurate manufacturing.